test/sql-server-cdc/mzcompose.py

```diff
@@ -234,7 +234,7 @@ def concurrent_updates(c: Composition) -> None:
         args=["--no-reset"],
         input=dedent(
             f"""
-            > SELECT COUNT(*) >= {update_rows+initial_rows}, MIN(id), MAX(id) >= {update_rows + update_id_offset - 1} FROM t1;
+            > SELECT COUNT(*) >= {update_rows + initial_rows}, MIN(id), MAX(id) >= {update_rows + update_id_offset - 1} FROM t1;
```

Comment:
formatting changes from ruff when my editor wrote the file to fix some merge conflicts. is it okay to leave them in?